pkg/healthcheck/healthcheck.go

```diff
@@ -55,6 +57,13 @@ type ServiceHealthChecker struct {
 	serviceName string
 }
 
+type PassiveHealthChecker struct {
+	mu          sync.Mutex
+	Failures    []time.Time
+	maxFail     int
+	failTimeout ptypes.Duration
+}
```

Comment:
Could you move this struct below, next to the `NewPassiveHealthChecker` function ?

Reply:
okay